index.html

```diff
@@ -5,15 +5,91 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>Karma</title>
     <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+    <script src="https://kit.fontawesome.com/c5012398d3.js" crossorigin="anonymous"></script>
+    
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <link rel="stylesheet" type="text/css" media="screen and (min-device-width: 800px)" href="css/over800.css" />
 </head>
 <body>
+    <nav>
+        <img class="nav-img" src="img\karma-logo.svg"   alt="Karma Logo">
+        <label for="toggle">&#9776;</label>
+        <input type="checkbox" id="toggle"/>
+        <div class="nav-a-container">
+            <a class="nav-a-first"><strong>Karma</strong></a>
+            <a class="nav-a">How it Works</a>
+            <a class="nav-a">Store</a>
+            <a class="nav-a">Blog</a>
+            <a class="nav-a">Help</a>
+            <a class="nav-a">Login</a>
+        </div>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+    </nav>
+
+    <article>
```

Comment:
Semantically, the header is a "top level" element in your document and so it doesn't make sense for it to be wrapped with <article> here - removing it shouldn't alter the layout